modules/@ergonode/ui/src/components/TextField/TextField.vue

```diff
@@ -35,6 +35,8 @@
                     autocomplete="on"
                     :disabled="disabled"
                     :aria-label="label || 'no description'"
+                    @keydown="onKeyDown"
```

Comment:
What's the purpose of introducing those listeners?

Reply:
disable page reloading on `enter` key. repairing a reported error

Reply:
Why is page reloading on key enter?

Reply:
To prevent page reloading here is the info: https://stackoverflow.com/questions/27364438/how-to-prevent-form-input-to-reload-a-page-when-receiving-enter-key/27364652

Submit event for form has to preventDefault behaviour.